src/packages/modules-operations/routes/index.tsx

```diff
@@ -38,5 +38,5 @@ const RootComponent = () => {
 	);
 };
 
-const Routes = () => <Route path="/" component={RootComponent} />;
+const Routes = () => <Route path="/"><RootComponent /></Route>;
```

Comment:
dans la doc pour la montée en react-router 5, ils disent de ne pas laisser un Route qui n'est pas à l'intérieur d'un Switch https://reactrouter.com/en/main/upgrading/v5#upgrade-to-react-router-v51

Il faut peut-être virer cette variable Routes et mettre un Switch dans la variable RootComponent qu'on exporte directement ?